friends/src/App.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Route, Redirect } from "react-router-dom";
+import { FriendsList } from "./components/FriendsList";
+import { AddFriendForm } from "./components/AddFriendForm";
+import { EditFriendForm } from "./components/EditFriendForm";
+import uuid from "uuid";
+import "./App.css";
+
+// Used a combination of this declaration and the spread operator throughout the file to minimize seeing this empty list:
+const emptyFriend = {
+  id: "",
+  name: "",
+  age: "",
+  email: "",
+  color: "",
+  favefood: "",
+  quotation: ""
+};
```

Comment:
very nice -- it's not necessary per se, but it saves so much time when excess form control is required that I really like this method :+1: